include/posix/pthread_sem.h

```diff
@@ -0,0 +1,17 @@
+/*
```

Comment:
In my system, these are in `semaphore.h`, not `pthread_sem.h`.  I'd move the declaration of `pthread_sem` to `semaphore.h` as well.